src/media/css/buttons.styl

```diff
@@ -174,6 +176,39 @@ commonprops($colour, $shadow) {
     }
 }
 
+.arrow-button-left,
+.arrow-button-right {
+    background-color: $action-positive;
+    background-repeat: no-repeat;
+    background-size: 15px 15px;
+    border: 1px solid $white;
+    border-radius: 50%;
+    color: transparent;
+    cursor: pointer;
+    font-size: 0;
+    height: $arrow-button-size;
+    transition: 150ms;
+    width: $arrow-button-size;
+
+    &:hover {
+        background-color: $action-positive-hover;
+    }
+
+    &:active {
+        background-color: $action-positive-tapped;
+    }
+}
+
+.arrow-button-left {
+    background-image: url(../img/pretty/arrow-l.svg);
+    background-position: 45% 50%;
+}
+
+.arrow-button-right {
```

Comment:
This could definitely be the same as `.arrow-button-left` but with `transform: rotate(180deg);`. Would save another image load. I'm not sure what would be better.